docs/content/latest/benchmark/tpcc.md

```diff
@@ -0,0 +1,101 @@
+---
+title: TPCC
+linkTitle: TPCC
+description: TPCC
+image: /images/section_icons/architecture/concepts.png
+headcontent: Benchmark YugabyteDB using TPCC.
+menu:
+  latest:
+    identifier: benchmark-tpcc
+    parent: benchmark
+    weight: 740
+aliases:
+  - /benchmark/tpcc/
+showAsideToc: True
+isTocNested: True
+---
+
+
+## Step 1. Install Java and Ant.
+
+## Step 2. Clone the TPCC repository.
+
+You can do this by running the following commands.
+
+```sh
+cd $HOME
+git clone https://github.com/yugabyte/tpcc.git
+cd tpcc
+```
+
+## Step 3. Compile the code
+
+You can compile the code using the following commands.
+
+```sh
+ant bootstrap
```

Comment:
Will prefer to move `cd $HOME/tpcc` to this step.

Reply:
Removed the compilation part since, we are now using the tar ball based approach.